src/components/cardSong/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
+import { getListSong } from "../../services/requetsToEndpoint";
+import Track from "../../models/track.inteface";
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import CardSongProps from "./cardSongProps.interface";
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** The use of inline styles within the `CardSongSearch` function (e.g., `style={{display:'inline-block', marginTop: '20px'}}`) is not recommended for maintainability and scalability. Consider using @emotion/styled or @mui/system for consistent theming and to leverage the full power of the CSS-in-JS solutions provided by the newly added dependencies.